controllers/get.article.controllers.js

```diff
@@ -18,4 +18,16 @@ exports.getAllArticles = (req, res, next) => {
     .catch(next);
 };
 
+exports.getComments = (req, res, next) => {
+  const objId = req.params;
+  const id = objId.article_id;
+  const endpoint = "/comments";
```

Comment:
this shouldn't be necessary as getComments will only be used if the path on line 18 of app.js is used which is already going to /comments